tests/e2e/workloads/jenkins/test_jenkins_node_drain.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import pytest
+
+from ocs_ci.framework.testlib import (
+    E2ETest, workloads
+)
+from ocs_ci.ocs.jenkins import Jenkins
+from ocs_ci.ocs import node
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def jenkins(request):
+
+    jenkins = Jenkins()
+
+    def teardown():
+        jenkins.cleanup()
+    request.addfinalizer(teardown)
+    return jenkins
+
+
+@workloads
+@pytest.mark.polarion_id("OCS-2175")
+class TestJenkinsNodeDrain(E2ETest):
+    """
+    Test running Jenkins and Node Reboot
+    """
+    @pytest.fixture()
+    def jenkins_setup(self, jenkins):
+        """
+        JENKINS test setup
+        """
+        # Deployment of jenkins
+        jenkins.setup_jenkins()
+
+    @pytest.mark.parametrize(
```

Comment:
There is only one arg and value, do we really need this parametrize here?

Reply:
I will add a new arg. 
The number of projects.